templates/image-builder.yml

```diff
@@ -261,3 +273,5 @@ parameters:
   - name: LOG_LEVEL
     value: "INFO"
     description: Main application log level (DEBUG, INFO, WARNING, ERROR, CRITICAL)
+  - name: RECOMMENDATIONS_URL
+    value: "https://rhel-image-builder-api-qa.apps.ext-waf.spoke.preprod.us-east-1.aws.paas.redhat.com/rhel-image-builder"
```

Comment:
Please put some dummy value here, do not expose internal URLs, the real endpoint will be put into app-interface.